src/component/feature/FeatureView/FeatureVariants/FeatureVariantsList/AddFeatureVariant/AddFeatureVariant.styles.ts

```diff
@@ -0,0 +1,28 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+export const useStyles = makeStyles(theme => ({
+    error: {
```

Comment:
Can you make this position absolute relative to it's parent so it doesn't cause a jump in the UI when the error message is displayed?